### ISSUE TITLE
Some Games won't load with DSi mode or TWL CPU (or both) enabled

### ISSUE THREAD

**Issue by zeroultima6** [ahezard/nds-bootstrap#716]
### Your console

Tell us which console you are using:

- [ ]  DS Phat/Lite
- [ ]  DSi (XL)
- [ ]  Old 3DS/2DS (XL)
- [ ]  New 3DS/2DS (XL)

### CFW and configuration

Tell us which CFW do you have:

DSi:

- [ ]  (No CFW) DSiWarehax (such as Flipnote Lenny, Sudokuhax, etc.)
- [ ]  (No CFW) Unlaunch only _[please specify version]_
- [ ]  Unlaunch+HiyaCFW _[please specify versions of both]_

3DS/2DS:

- [ ]  Luma3DS
- [ ]  Rei-Six
- [ ]  Other/Older CFW [specify]

### nds-bootstrap version

Tell us which versions you are using:

- [ ]  Release _[0.22.0]_
- [ ]  Nightly _[f8da9b8]_

### Your device

Tell us which card you are using:

- [ ]  SD Card
- [ ]  (Slot-1) Flashcard

### If this issue only affects a certain game, which game is it?

Game goes here if applicable

Radiant Historia

### Expected behaviour

Tell us what should happen

Radiant Historia: works as intended

### Actual behaviour

Tell us what happens instead

Radiant Historia: glitch out and freeze on boot

### Steps to reproduce

1.
2.
3.

### Other notes
The games are non-dsi enhanced BTW, might add more if any

**Comment by dubbz82** [comment]
This is a non-issue, at least for most of the games that have problems, as they were never designed to run at TWL CPU speeds.  It's there as an option for people to play around with and see what can take advantage of the added speed bump, but the same thing would happen if you were to somehow force the cpu of a DSI to stay running at TWL speeds while booting a physical cart.

**Comment by dubbz82** [comment]
My initial thought on this is perhaps specific games may be blacklisted from running under DSi Mode and/or TWL CPU speeds to prevent these sorts of problems, however it seems unlikely that you'll be able to play them with the settings as they are, as they're problems that are related to the games themselves not liking the cpu speed

**Comment by zeroultima6** [comment]
If that is true, what some of the games are blacklisted with those issces above?

**Comment by dubbz82** [comment]
Blacklist isn't currently a thing for this, was more of just a thought I had to possibly address it and to prevent it from happening in the future.

**Comment by zeroultima6** [comment]
Makes sense, those games work well before with those options on and now they have to have those options on specifically for it to work but hey it is what it is.

**Comment by vaguerant** [comment]
Just adding a note here that the graphics in The Wizard of Oz - Beyond the Yellow Brick Road (USA)/Riz-Zoawd (Japan) turn to a flickering mess if you run them at TWL clock-speed, they should be added to any prospective blacklist should such a feature be implemented.

**Comment by NightYoshi370** [comment]
As this is not an issue within bootstrap itself and rather within the games, I see no reason for this issue to remain open.

It's why MelonDSi switches to use the DS BIOS for when launching DS titles
